src/lib/render.ts

```diff
@@ -51,6 +54,17 @@ const renderDistractorBlocks = (settings: ParsonsSettings): Cash => {
   return distractorBlockContainer
 }
 
+const getCodeBlocks = (code: string, isDistractors: boolean): string => {
```

Comment:
Instead of filtering code twice you can return tuple() here with ```[codeBlocks: string, distractors: string]```
use ```lines.foreach``` with ```regex.exec(line)``` and get matches from result